README.md

```diff
@@ -195,6 +195,14 @@ metrics = require('metrics')
   * `num` Value to put in histogram (number).
   * `label_pairs` Table containing label names as keys, label values as values (table). New value is observed by all internal counters with these labels specified.
 
+#### `histogram_obj:observe_latency(label_pairs, fn, ...)`
+  Measure latency of function call
+  * `label_pairs` either table with labels or function to generate labels
+  * `fn` function for pcall to instrument
+  * `...` args for function `fn`
+
+  Return results in format `res, err` of fn call or reraise error
```

Comment:
It looks like too artificial restriction. Seems you could implement something like that https://github.com/tarantool/tarantool/blob/fe6b7d3c03dbe87e7f97893bf255c2e1ebf86fe1/src/box/lua/schema.lua#L349

Reply:
Fixed